.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Unit tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  project-list:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo apt-get -y install jq
+      - name: Get Project List
+        id: list-projects
+        run: echo "::set-output name=projects::$(find -wholename "./*/pom.xml" -exec dirname {} \; | jq --compact-output --raw-input [inputs])"
```

Comment:
Beautiful :smile:

Reply:
I'm so proud of that :laughing:

Reply:
TIL: `mvn -Dexec.executable='echo' -Dexec.args='${project.artifactId}' exec:exec -q` :D https://stackoverflow.com/questions/3662291/how-to-list-active-sub-modules-in-a-maven-project

Reply:
https://stackoverflow.com/questions/1114026/maven-modules-building-a-single-specific-module

> Note that you need to use a colon if you are referencing an artifactId which differs from the directory name:

> mvn install -pl :B -am

Reply:
Enough exploding head for today :laughing: